stdlib/_interpreters.pyi

```diff
@@ -0,0 +1,48 @@
+import types
+from collections.abc import Callable, Mapping
+from typing import Final, Literal, SupportsIndex, final
+from typing_extensions import TypeAlias
+
+_Configs: TypeAlias = Literal["default", "isolated", "legacy", "empty"]
+
+class InterpreterError(Exception): ...
+class InterpreterNotFoundError(InterpreterError): ...
+class NotShareableError(Exception): ...
+
+# Not-instantiable, so mark as final.
```

Comment:
That's not what final means; it refers to subclassing, and this class can in fact be subclassed.

Reply:
Yes, this is mainly because I don't see a reason why it should be subclassed. There was a similar situation in the `_interpchannels` module, where @AlexWaygood agreed it didn't really make sense either, [so just put `final` on it.](https://github.com/python/typeshed/pull/12199#discussion_r1652331296)

Happy to remove it, but probably makes sense to change that as well though.

Reply:
The comment definitely isn't clear though, so apologies there.

Reply:
The flags on the type explicitly include BASETYPE, so I'm pretty sure it's meant to be subclassable.

Reply:
Makes sense! https://github.com/python/typeshed/pull/12230/commits/9cb5a5915014601bd00812adca7537bf1425b2cd